lib/foreman_maintain/runner.rb

```diff
@@ -142,13 +143,16 @@ def post_step_decisions(scenario, execution)
 
     # rubocop:disable  Metrics/MethodLength
     def ask_about_offered_steps(step)
-      if assumeyes? && rerun_check?(step)
```

Comment:
Currently it is checking `rerun_check?` along with `assumeyes?` so what is expected when there is no any next steps?

Reply:
@kgaikwad if there is no any next steps then it is expected that check will fail.